api/src/main/java/org/commonjava/maven/galley/model/Transfer.java

```diff
@@ -513,4 +519,14 @@ public void setResource( ConcreteResource resource )
     {
         this.resource = resource;
     }
-}
+
+    public void setGeneratedFromTarget( Set<Transfer> generatedFromTarget )
```

Comment:
Can we handle this split in the Indy code? It seems strange to keep state about "real" Transfers that were split out of this Transfer